### PR TITLE
Adding Coats to the Staff Officer vendor

### PR DESCRIPTION
**Pull request by theselfish**
# About the pull request

Title.
image
I am willing to toss the Tanker Jacket, as much as I miss it.
I also added the uniform to their required gear for the sake of consistency. 
# Explain why it's good for the game

It gives the SO more options for clothing, as it's cold on the ship.
I've also removed the free helmet they could get, for some odd reason.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: 
add: SOs may now get coats in their vendor.
/:cl:

### CODE REVIEW

**Comment by theselfish** [comment]
@harryob Since no maintainer was pinged.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Undo the helmet/hat change. Feel free to give them a *regular* cmcap as an option.

Uniform change fine.

Only service jacket choosable. Dress shouldn't be used regularly and tanker isn't for SOs.

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 1]
```diff
@@ -11,11 +11,17 @@
 
 GLOBAL_LIST_INIT(cm_vending_clothing_staff_officer, list(
 		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
+		list("Uniform", 0, /obj/item/clothing/under/marine/officer/bridge, MARINE_CAN_BUY_UNIFORM, VENDOR_ITEM_MANDATORY),
 		list("Boots", 0, /obj/item/clothing/shoes/marine/knife, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),
 		list("Headset", 0, /obj/item/device/radio/headset/almayer/mcom, MARINE_CAN_BUY_EAR, VENDOR_ITEM_MANDATORY),
-		list("Helmet", 0, /obj/item/clothing/head/helmet/marine/MP/SO, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_MANDATORY),
+		list("Hat", 0, /obj/item/clothing/head/cmcap/ro, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_MANDATORY),
```
Comment (theselfish) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:17]:
```suggestion
		list("Helmet", 0, /obj/item/clothing/head/helmet/marine/MP/SO, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_MANDATORY),
```

**Comment by theselfish** [comment]
> Undo the helmet/hat change. Feel free to give them a _regular_ cmcap as an option.
> 
> Uniform change fine.
> 
> Only service jacket choosable. Dress shouldn't be used regularly and tanker isn't for SOs.

The /ro cap is the same as the one they spawn with.
But, doing it as I type.

**Comment by theselfish** [comment]
The RO's hat is, confusingly under /req.

**Comment by theselfish** [comment]
@morrowwolf Sorted.

**Review by harryob — APPROVED** [review]

**Pull request merged**